src/uucore/src/lib/features/utmpx.rs

```diff
@@ -192,13 +192,9 @@ impl Utmpx {
     }
     /// A.K.A. ut.ut_tv
     pub fn login_time(&self) -> time::OffsetDateTime {
-        #[cfg(all(not(target_os = "freebsd"), not(target_vendor = "apple")))]
-        let ts_nanos: i128 = (self.inner.ut_tv.tv_sec as i64 * 1_000_000_000_i64
-            + self.inner.ut_tv.tv_usec as i64 * 1_000_i64)
-            .into();
-        #[cfg(any(target_os = "freebsd", target_vendor = "apple"))]
-        let ts_nanos: i128 = (self.inner.ut_tv.tv_sec * 1_000_000_000_i64
-            + self.inner.ut_tv.tv_usec as i64 * 1_000_i64)
+        #[allow(clippy::useless_conversion)]
```

Comment:
Seems like it's called `clippy::unnecessary-cast`? That's what the CI complains about at least.

Reply:
Yep I fix it as soon as I am at home :) alternatively, I can create two temporary i64: sec and usec and use them. I guess this would coerce the values to i64 if necessary and I don't have to ignore something. But clippy might just drop another warning 😁